### ISSUE TITLE
Cloud-init 'runcmd' doesn't work on Linux VM

### ISSUE THREAD

**Issue by fatmcgav** [fog/fog-vsphere#96]
#34 added support for running the 'runcmd' section from cloud-init data. 

However those changes don't work on Linux VMs', as they use the 'SysPrep' functionality... 

Would be really good to be able to support on Linux VMs' aswell...

**Comment by angystardust** [comment]
As mentioned by the foreman devs here (https://groups.google.com/d/topic/foreman-dev/n1QkZF1QVxU/discussion), I'm afraid that the "runcmd" support on Linux is limited by the vmware CustomSpecs...

**Comment by lzap** [comment]
Yeah, that's the case. Wondering why they designed it this way, perhaps for Windows compatibility. Not sure.

**Comment by MrMEEE** [comment]
Is it the same case for "write_files"?
